spec/integration_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'rails_helper'
+
+# RSpec.describe 'Siggin Up', type: :feature do
+#   scenario 'Sign up with valid inputs' do
+#     visit new_user_registration_path
+#     fill_in 'Name', with: 'Foo Bar'
+#     fill_in 'Email', with: 'foo@bar.com'
+#     fill_in 'Password', with: '12345678'
+#     fill_in 'Password confirmation', with: '12345678'
+#     click_on 'Sign up'
+#     sleep(3)
+#     visit root_path
+#     expect(page).to have_content('Foo Bar')
+#   end
+
+#   scenario 'Sign up with invalid inputs' do
+#     visit new_user_registration_path
+#     fill_in 'Name', with: ''
+#     fill_in 'Email', with: 'foo@bar.com'
+#     fill_in 'Password', with: '12345678'
+#     fill_in 'Password confirmation', with: '12345678'
+#     click_on 'Sign up'
+#     sleep(3)
+#     visit root_path
+#     expect(page).to_not have_content('Foo Bar')
+#   end
+# end
+
+# RSpec.describe 'Sign In', type: :feature do
+#   let(:user) { User.create(name: 'Foo Bar', email: 'foo@bar.com', password: '12345678') }
+#   scenario 'Log in with valid inputs' do
+#     visit new_user_session_path
+#     fill_in 'Email', with: user.email
+#     fill_in 'Password', with: user.password
+#     click_on 'Log in'
+#     sleep(3)
+#     expect(page).to have_content('Foo Bar')
+#   end
+
+#   scenario 'Log in with invalid inputs' do
+#     visit new_user_session_path
+#     fill_in 'Email', with: 'Banana'
+#     fill_in 'Password', with: user.password
+#     click_on 'Log in'
+#     sleep(3)
+#     expect(page).to_not have_content('Foo Bar')
+#   end
+# end
+
+# RSpec.describe 'Loggin Out', type: :feature do
+#   let(:user) { User.create(name: 'Foo Bar', email: 'foo@bar.com', password: '12345678') }
+#   scenario 'Log Out' do
+#     visit new_user_session_path
+#     fill_in 'Email', with: user.email
+#     fill_in 'Password', with: user.password
+#     click_on 'Log in'
+#     sleep(3)
+#     click_on 'Sign Out'
+#     sleep(3)
+#     visit root_path
+#     expect(page).to have_content('Sign In')
+#   end
+# end
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
See this [post](https://stackoverflow.com/questions/13130859/inline-comments-in-ruby/13130900#:~:text=No%2C%20Ruby%20does%20not%20have,comment%20out%20the%20one%20row), about why comments are discouraged.